tests/msgpack.tests/VS/msgpack.tests.VS.csproj

```diff
@@ -37,15 +37,88 @@
     <None Include="project.json" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\**\*.cs">
-      <Link>%(RecursiveDir)%(Filename)%(Extension)</Link>
+    <Compile Include="..\Properties\AssemblyInfo.cs">
```

Comment:
As I remember, I didn't add any files using GUI. And unfortunately I don't have ideas why single wildcard <Compile> replaced with per-file <Compile>.